app/controllers/comments_controller.rb

```diff
@@ -1,10 +1,28 @@
 class CommentsController < ApplicationController
+  def index
+    user = User.find(params[:user_id])
+    post = user.posts.find(params[:post_id])
+    comments = post.comments
+    render json: comments
+  end
```

Comment:
- [ ] The api endpoint for listing all the comments for a post is not working at the moment. Kindly fix it.

### Error
![comments-for-a-post](https://user-images.githubusercontent.com/26597432/232091549-d0d09120-a56f-4f01-945c-36e9bc7e4bf1.png)

### Project Requirements
![list-comment](https://user-images.githubusercontent.com/26597432/232093300-6dda0e45-5cb5-41e2-a915-72c5109a726c.png)